handler_test.go

```diff
@@ -297,6 +336,6 @@ func BenchmarkHandlerDispatch(b *testing.B) {
 
 type nopResponseWriter struct{}
 
-func (nopResponseWriter) Header() http.Header       { return nil }
+func (nopResponseWriter) Header() http.Header       { return http.Header{} }
```

Comment:
ℹ️ We now set headers. Benchmark would panic without this.

Reply:
The comment is helpful; thanks!